packages/qwik/src/core/reactive-primitives/internal-api.ts

```diff
@@ -57,7 +57,7 @@ export const _wrapProp = <T extends Record<any, any>, P extends keyof T>(...args
     }
   }
   // We need to forward the access to the original object
```

Comment:
this comment is no longer correct, it should be something like "not a reactive object, just access it"